polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -356,6 +357,59 @@ ValueCategory MLIRScanner::VisitPredefinedExpr(clang::PredefinedExpr *Expr) {
 }
 
 ValueCategory MLIRScanner::VisitInitListExpr(clang::InitListExpr *Expr) {
+  LLVM_DEBUG({
+    llvm::dbgs() << "VisitInitListExpr: ";
+    Expr->dump();
+    llvm::dbgs() << "\n";
+  });
+
+  assert(!Expr->hadArrayRangeDesignator() && "Unsupported");
+
+  const auto Loc = getMLIRLocation(Expr->getExprLoc());
```

Comment:
Can we create a utility function (a static function on a private member function of  MLIRScanner) to put this code in ?

Reply:
Created a private member function.